src/LabLog.Domain/Entities/Room.cs

```diff
@@ -17,9 +17,10 @@ public static Room Create(Action<ILabEvent> eventHandler)
         {
             var room = new Room(eventHandler);
             room.Id = Guid.NewGuid();
-            room._eventHandler(new LabEvent<RoomCreatedEvent>(room.Id, 
-                ++room.Version,
-                new RoomCreatedEvent()));
+            var e = LabEvent.Create(room.Id, 
+                ++room.Version, new RoomCreatedEvent());
+            e.SetEventBody(new RoomCreatedEvent());

```

Comment:
e is created with the return from LabEvent.Create, which is passed some stuff AND a new RoomCreatedEvent.
Then e.SetEventBody is passed ANOTHER RoomCreatedEvent.
Why?

Reply:
Oops :)